src/components/postgresConnect.ts

```diff
@@ -450,7 +450,7 @@ class PostgresDB {
     }
   }
 
-  public getCurrentStatusHistory = async (applicationId: number) => {
+  public getApplicationCurrentStatusHistory = async (applicationId: number) => {
```

Comment:
Assume you did a global search for anywhere else this might be called?

Reply:
Yeah i did a global replace